specification/resources/resource-manager/Microsoft.Authorization/preview/2023-03-01-preview/privateLinks.json

```diff
@@ -0,0 +1,834 @@
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Authorization/resourceManagementPrivateLinks/{rmplName}": {
+      "put": {
+        "tags": [
+          "ResourceManagementPrivateLink"
+        ],
+        "operationId": "ResourceManagementPrivateLink_Put",
+        "description": "Create a resource management group private link.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "minLength": 1,
+            "maxLength": 90
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceManagementPrivateLinkParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ResourceManagementPrivateLinkLocation"
+            },
+            "description": "The region to create the Resource Management private link."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Returns information about the Resource Management Private Link.",
+            "schema": {
+              "$ref": "#/definitions/ResourceManagementPrivateLink"
+            }
+          },
+          "201": {
+            "description": "Created. Returns information about the Resource Management Private Link.",
+            "schema": {
+              "$ref": "#/definitions/ResourceManagementPrivateLink"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create Resource Management Private Link.": {
+            "$ref": "./examples/PutResourceManagementPrivateLink.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "ResourceManagementPrivateLink"
+        ],
+        "operationId": "ResourceManagementPrivateLink_Get",
+        "description": "Get a resource management private link(resource-level).",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "minLength": 1,
+            "maxLength": 90
+          },
```

Comment:
Use resource group name from common-types, instead of defining it here: [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5).